ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable-4.17-upgrade-from-stable-4.17.yaml

```diff
@@ -161,8 +161,8 @@ tests:
     test:
     - chain: openshift-upgrade-qe-sanity-disconnected
     workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-private-cco-manual-workload-identity
-- as: gcp-ipi-proxy-tp-f28
```

Comment:
We run this job with a OTA test case, so suggest to keep it on f28.

Reply:
done